### DISCUSSION TITLE
changing wifi network parameters

### DISCUSSION THREAD

**Discussion by FireSimpson** [BitMaker-hub/NerdMiner_v2#318] [Q&A]
ESP32-wroom-32

Due to the move, it became necessary to change the network parameters. Through the phone and laptop the NerdMinerAP network cannot be found. After rewriting the code again, there is also no connection with the device through the jack (it is not visible), but at the same time it continues to work with the old network parameters.

Tell me how to change the network parameters?

(when recording via BIN, you can add a file with parameters)

**Comment by MyOwn2C** [comment]
Did you try this?
<img width="826" alt="image" src="https://github.com/BitMaker-hub/NerdMiner_v2/assets/158797249/d8a14c66-7ed2-4532-bfeb-acfe188a534c">

**Reply by sany3001** [reply]
I am missing in documentation how to work with button-less but touch screen devices.

**Comment by FireSimpson** [comment]
> I have a block with several ESP32-wroom 32, but unfortunately I tried all possible options.

**Reply by BitMaker-hub** [reply]
In that case try to delete all flash on the device and reflash them again. I'm sorry there is no other current option

**Reply by BitMaker-hub** [reply]
You can delete all flash here : https://espressif.github.io/esptool-js/
